zdf_2_ply_without_zivid.py

```diff
@@ -1,10 +1,10 @@
 """
-This example shows how to convert a Zivid point cloud from a .ZDF file format
-to a .PLY file format.
+Convert ZDF point cloud to PLY file format without Zivid Software.
```

Comment:
without zivid software?

Reply:
The point is that you don't need to have Zivid installed to work with ZDF files since it is based on a standard file format.